artifacts/otg.proto

```diff
@@ -3889,6 +3893,28 @@ message BgpGracefulRestart {
   optional uint32 stale_time = 4;
 }
 
+// Ordered BGP Updates ( including both Advertise and Withdraws ) to be sent to the
+// peer after the BGP session is established.
+message BgpUpdateReplay {
+
+  // Array of ordered BGP Updates ( including both Advertise and Withdraws ) to be sent
+  // to the peer after the BGP session is established.
+  repeated BgpOneUpdateReplay updates = 1;
+}
+
+// Specification of one BGP Update to be sent to the BGP peer.
+message BgpOneUpdateReplay {
+
+  // Bytes to be sent to peer after the BGP Update Header. The Update Header will always
+  // have the initial 16 bytes containing  Marker bytes, 2 bytes containing the Length
+  // and 1 byte containing the Type.The string MUST contain sequence of valid hex bytes.
+  // The bytes specified should be appended to  the Update message to be sent to the peer
+  // after the fixed 19 bytes above.  The maximum length of 4077 is obtained from the
```

Comment:
Since we are tacking about maximum, we should talk about minimum also.

Reply:
I specifically mentioned the logic for maximum since without this 4077 might appear to be an arbitrary max. Either way, I have added that the minimum length of the input byte stream should be 1 as per previous comment since we are allowing the user to send whatever he/she wants upto max length.